apps/docs/pages/guides/auth/auth-helpers/remix.mdx

```diff
@@ -757,7 +757,7 @@ export default function Index() {
 
 </Tabs>
 
-> Ensure you have [enabled replication](https://supabase.com/dashboard/project/_/database/replication) on the table you are subscribing to.
+> Ensure you have [enabled replication](https://supabase.com/dashboard/project/_/database/publication) on the table you are subscribing to.
```

Comment:
Do we also want to rename it to `publication` in the text? Could be confusing if dashboard calls it `publication` and docs calls it `replication`